styled.css

```diff
@@ -0,0 +1,299 @@
+
+.inerWrapperRoadMap {
+  display: flex;
+  justify-content: center;
+}
+
+.roadMApContainer {
+  max-width: 1170px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.roadMapTitle {
+  margin: 64px;
+  color: #c6cdd9;
+  text-align: center;
+  font: sans-serif;
+  font-size: 48px;
+}
+.roadMapWrapper {
+  margin-top: 40px;
+}
+.swiperWrapper {
+  display: flex;
+  justify-content: center;
+  margin: 62px;
+}
+
+.swiperWrapperText {
+  max-width: 172px;
+  width: 100%;
+  font-size: 18px;
+}
+.timeStampText {
+  display: flex;
+  width: 100%;
+  margin-bottom: 20px;
+  flex-direction: column;
+  color: #fff;
+}
+.statusWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 10px 5px;
+  position: relative;
+}
+.statusWrapper::before {
+  content: "";
+  position: absolute;
+  width: 184px;
+  height: 23px;
+  background-color: blue;
+  top: -8px;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+  border-radius: 25px 25px 25px 25px;
+}
+.status {
+  width: 100%;
+  background: #191c29;
+  border-radius: 14px;
+  margin: 5px;
+  padding: 20px;
+  font-family: NexaLight;
+  text-align: left;
+  position: relative;
+  top: 15px;
+  font-size: 18px;
+  font-weight: 300;
+  line-height: 30px;
+  letter-spacing: normal;
+  color: white;
+}
+.status::before {
+  content: "";
+  position: absolute;
+  width: 16px;
+  height: 16px;
+  background-color: #fff;
+  border-radius: 25px;
+  top: -35px;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+}
+
+
+.backgroundImagefeature {
+  position: absolute;
+  background-size: contain;
+  max-width: 390px;
+  width: 100%;
+  height: 390px;
+  left: 0;
+  top: 473px;
+  z-index: -1;
+}
+
+.shieldImage {
+  width: 319px;
+}
+
+.roadmapDate {
+  text-align: center;
+}
+.roadmaproadmapDateMobile {
+  display: none;
+}
+
+@media (max-width: 991px) {
+  .heading {
+    visibility: hidden;
+    color: transparent;
+
+  }
+
+  .button {
+    padding: 8px 16px;
+    font-size: 13px;
+    width: max-content;
+  }
+  .description {
+    color: #bdc2cb;
+    font-size: 12px;
+    line-height: 20px;
+    margin: 0 0 15px;
+    font-family: NexaLight;
+  }
+
+  .leftDiv {
+    margin: 0 20px 0 0;
+    max-width: 130px;
+    width: 50%;
+    text-align: left;
+  }
+
+  .TitleMobile {
+    visibility: visible;
+    font-family: NexaBold;
+    font-size: 28px;
+    font-weight: 700;
+    line-height: 33px;
+    letter-spacing: normal;
+    color: #c6cddd;
+    margin: 0 0 30px;
+  }
+
+  /* --second section ----- */
+  .swiperWrapper {
+    overflow-x: scroll;
+    justify-content: flex-start;
+    margin: 0px;
+  }
+
+  .swiperWrapperText {
+    min-width: 187px;
+  }
+
+  .roadmapDate {
+    display: none;
+  }
+  .statusWrapper::before {
+    height: 8px;
+    top: 0;
+    width: 194px;
+  }
+
+  .status::before {
+    left: 36px;
+    margin: 0;
+    width: 8px;
+    height: 8px;
+    background-color: #fff;
+    border-radius: 25px;
+    border: none;
+    top: -15px;
+  }
+
+  .roadmaproadmapDateMobile {
```

Comment:
roadmaproadmapDateMobile >> roadmapDate

here for mobile give different class mobileShow or mobileHide and use common for all @PRABgit01